packages/waku/src/lib/utils/merge-vite-config.ts

```diff
@@ -11,6 +11,7 @@ export async function mergeUserViteConfig(config: UserConfig) {
   const mergedViteConfig = mergeViteConfig(
     {
       ...resolvedViteConfig,
+      configFile: false,
```

Comment:
So, config file is already resolved in `resolvedViteConfig`, right?
I mean we still support custom `vite.config.ts`. cc @Aslemammad